programs/protocol_event/src/state/event.rs

```diff
@@ -0,0 +1,36 @@
+use anchor_lang::prelude::*;
+
+#[account]
+pub struct Event {
+    pub authority: Pubkey,
+    pub payer: Pubkey,
+
+    pub slug: String, // event identifier e.g. LAFCvLAG@2021-08-28
```

Comment:
Not 100% happy with `slug`naming  here as we don't use that anywhere else, but it is in the doc proposal and other APIs (e.g. smarkets) use it